networkx/generators/tests/test_random_graphs.py

```diff
@@ -5,18 +5,105 @@
 import pytest
 
 
+class TestGnpRandomGraphs:
```

Comment:
I'd prefer to get rid of the outer classes entirely - they slow down test collection and don't really add anything (once the tests are also parametrized over generators)